src/app/views/interactives/components/FooterComponent.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+
+export class FooterComponent extends Component {
+    render() {
```

Comment:
Is this a new footer? Is this being added to all Florence pages?
If not shouldn't we be using the same one as other parts of the app?

Reply:
in fact, there is no footer in florence yet, so its a new one and will be added on all florence pages as per figma